src/EventStore.Core.XUnit.Tests/Startup/StartupStatusTrackerTests.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Linq;
+using EventStore.Core.Data;
+using EventStore.Core.Services.VNode;
+using EventStore.Core.Telemetry;
+using EventStore.Core.XUnit.Tests.Telemetry;
+using Xunit;
+
+namespace EventStore.Core.XUnit.Tests.Startup {
+	public class StartupStatusTrackerTests {
+		private readonly FakeClock _clock = new();
+		private readonly StatusMetric _metric;
+		private readonly StartupStatusTracker _sut;
+
+		public StartupStatusTrackerTests() {
+			_metric = new StatusMetric(
+				new Meter($"Eventstore.Core.XUnit.Tests.{nameof(StartupStatusTrackerTests)}"),
```

Comment:
```suggestion
				new Meter($"{typeof(StartupStatusTrackerTests)}"),
```

i think i changed to doing it like this in my own tests somewhere along the way

Reply:
Done